Detectors/ITSMFT/ITS/macros/test/CheckLookUp.C

```diff
@@ -58,8 +59,16 @@ void CheckLookUp(std::string clusfile = "o2clus_its_comp.root", std::string dict
   // Clusters
   TFile* file1 = TFile::Open(clusfile.data());
   TTree* clusTree = (TTree*)gFile->Get("o2sim");
-  std::vector<Cluster>* clusArr = nullptr;
-  clusTree->SetBranchAddress("ITSCluster", &clusArr);
+  std::vector<CompClusterExt>* clusArr = nullptr;
+  clusTree->SetBranchAddress("ITSClusterComp", &clusArr);
+  std::vector<unsigned char>* patternsPtr = nullptr;
+  auto pattBranch = clusTree->GetBranch("ITSClusterPatt");
+  if (pattBranch) {
+    pattBranch->SetAddress(&patternsPtr);
+  } else {
+    std::cout << "Missing branch with patterns." << std::endl;
+    return;
+  }
```

Comment:
@shahor02 This macro does not make much sense for compact clusters.  We should probably leave it as it was.
Switching here  to compact clusters will break the compilation, because the compact clusters do not have several
functions that used below.